lldb/tools/lldb-dap/src-ts/debug-session-tracker.ts

```diff
@@ -62,8 +63,12 @@ export class DebugSessionTracker
   createDebugAdapterTracker(
     session: vscode.DebugSession,
   ): vscode.ProviderResult<vscode.DebugAdapterTracker> {
+    this.logger.info(`Starting debug session "${session.name}"`);
+    let stopping = false;
     return {
+      onError: (error) => !stopping && this.logger.error(error), // Can throw benign read errors when shutting down
```

Comment:
Nit: missing period. 

```suggestion
      onError: (error) => !stopping && this.logger.error(error), // Can throw benign read errors when shutting down.
```